Cannot install using CMake because schelp path is not correct

Hey Nathan. Amazing job once again. I am trying to compile and install using CMake but it looks like you haven't updated the CMakelists.txt file to include the correct path to the help file: 


```
CMake Error at cmake_install.cmake:82 (file):
  file INSTALL cannot find
  "/home/mads/code/arch-packages/supercollider-plugins/supercollider-cd-skip-git/src/supercollider-cd-skip-git/plugins/CDSkip/CDSkip.schelp":
  No such file or directory.
```

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -76,7 +76,7 @@ set(CDSkip_sc_files
     plugins/CDSkip/CDSkip.sc
 )
 set(CDSkip_schelp_files
-    plugins/CDSkip/CDSkip.schelp
+    HelpSource/Classes/CDSkip.schelp
 )
 
 sc_add_server_plugin(

